### ISSUE TITLE
handle the parameter in the STANAG messages

### ISSUE THREAD

**Issue by frizman21** [albersgroup/picante#46]

**Comment by frizman21** [comment]
@Reliquia33ad Do you remember which parameter we were talking about. I've lost the context on this issue.

**Comment by Reliquia33ad** [comment]
No.  I do not.   The only parameter that I thought we discussed was messageTypeId and that we wanted it to be a String rather than an int.  And that it would be called messageTypeId instead of messageId. The implementation at our noon review was as an int,  since introduction of that field was part of the existing enhancement, I took the suggestion to be clarification of requirements and incorporated the work.
